packages/build-config/package.json

```diff
@@ -0,0 +1,88 @@
+{
+	"name": "@wordpress/build-config",
+	"version": "1.0.0",
+	"description": "WordPress JavaScript build configuration.",
+	"author": "The WordPress Contributors",
+	"license": "GPL-2.0-or-later",
+	"keywords": [
+		"wordpress",
+		"build",
+		"config"
+	],
+	"homepage": "https://github.com/WordPress/gutenberg/tree/master/packages/build-config/README.md",
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/WordPress/gutenberg.git"
+	},
+	"bugs": {
+		"url": "https://github.com/WordPress/gutenberg/issues"
+	},
+	"engines": {
+		"node": ">=8",
+		"npm": ">=6.0.0"
+	},
+	"files": [
+		"build",
```

Comment:
I don't think there are build files generated for this package. They are generated only when you use `src` folder.

Reply:
fixed acc77a4c8e3d7d3511d79e2435c88d1f2a5b1cb9